pkg/util/timeutil/BUILD.bazel

```diff
@@ -49,3 +50,21 @@ go_test(
         "@com_github_stretchr_testify//require",
     ],
 )
+
+genrule(
+    name = "gen-lowercase-timezones-go",
+    srcs = ["//pkg/util/timeutil/lowercase"],
+    outs = ["lowercase_timezones.go"],
+    cmd = """
+    env GOROOT=`dirname $(location @go_sdk//:ROOT)` \
+    env PATH=`dirname $(location @go_sdk//:bin/go)`:`dirname $(location @com_github_cockroachdb_crlfmt//:crlfmt)`:$$PATH HOME=$(GENDIR) \
+    $(location //pkg/util/timeutil/lowercase) \
+    --filename=$(location lowercase_timezones.go)  
+       """,
+    tools = [
+        "@com_github_cockroachdb_crlfmt//:crlfmt",
+        "@go_sdk//:ROOT",
+        "@go_sdk//:bin/go",
```

Comment:
As I said before, I really don't want to link the entire `go` SDK into these `genrules`. Why did we regress on this again? What problem are you trying to solve by doing this?

Reply:
As code `pkg/util/timeutil/lowercase/main.go` is using `runtime.GOROOT()` we need to tell Bazel which is the path for GOROOT, that is why i found that solution, even that we are already using a `go_binary` `//pkg/util/timeutil/lowercase`

Not sure if you have a better idea on how to handle this.

The other ones works pretty good only with the binary but the main problem in here is that this file is using a file and a run command from GOROOT

Reply:
This still needs to be fixed.

Reply:
Again, this still needs to be fixed.